### ISSUE TITLE
marking menu doesn't appear in 2018.1

### ISSUE THREAD

**Issue by gkaizer** [DavideAlidosi/May9#6]
Hi, I just installed it in a fresh new 2018 folder to test it, but when I hit Z and then MMB, nothing happens.Hitting Z alone either, and changing hotkey neither.
What I can I do?
I'm on Maya 2018.1 Educational Standalone on Windows 10

**Comment by DavideAlidosi** [comment]
Hi Andrea,
actually I don't have 2018.1, and for fix this bug I need it.
Tnx,

**Comment by gkaizer** [comment]
I don't think there are any big differences between 2018 and 2018.1, as it fixed just some bugs and improved performance.BTW, I did try to use it before upgrading it to 2018.1, and it behaved like that, so I upgraded and tested again but with no luck

**Comment by gkaizer** [comment]
I commented out the "except Exception:   pass" code from the da_MMtoKey_press runtime command in the hotkey editor to see what's causing it, every time I hit Z an error is thrown and printed to the script input window:
"# Error: line 1: invalid syntax # "

**Comment by DavideAlidosi** [comment]
Ok, try to check:
*	If workspace is set to May9 Pro
*	If under Windows > Settings/Preference > Hotkey is set to May9 Pro

**Comment by gkaizer** [comment]
nope, they're already like that, it's a fresh install, the same 2018 from github, no mod

**Comment by DavideAlidosi** [comment]
The syntax error is related to runtime itself, try to leave the call code only:

MMtoKey.pressSelected(ctl=False, alt=False, sh=False)

**Comment by gkaizer** [comment]
ok, for the time being I'm gonna uninstall it, it's useless like so, hope somebody else will experience that and hopefully fix it

**Comment by DavideAlidosi** [comment]
You can make this try, to run this Python:
MMtoKey.ui()
And then:
1) Tools > Hotkey maker
2) Give a name
3) Add z as hotkey
4) Press on create

Anyway is strange, on what os you are?

**Comment by gkaizer** [comment]
windows 10

**Comment by DavideAlidosi** [comment]
What I wan't to understand is if MMtoKey work, have you try to use:
`MMtoKey.ui()`
Or CTRL + ALT + SHIFT + Z ?

**Comment by DavideAlidosi** [comment]
I've installed Maya 2018.1 on my Windows 10 but everything is ok.

**Comment by gkaizer** [comment]
Yes tried both, nothing changed, don't know whether there is another script or plugin in conflict preventing it to being called, i'll look into it

**Comment by DavideAlidosi** [comment]
I've commit MMtoKey loaded as Plug-in, can please download again May9 from Git and install in a clean directory? Hope it fix.

**Comment by gkaizer** [comment]
ok I'll try, thanks for your support
